server/routes/gcp/service_accounts.py

```
@@ -0,0 +1,329 @@
+    # 2. Best-effort enumeration of all accessible projects
+    accessible: List[str] = []
+    try:
+        resp = crm.projects().list().execute()
+        for proj in resp.get("projects") or []:
+            pid = proj.get("projectId")
+            if not pid:
+                continue
+            if (proj.get("lifecycleState") or "ACTIVE") != "ACTIVE":
+                continue
+            accessible.append(pid)
+    except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)